src/content/docs/ai-monitoring/customize-agent-ai-monitoring.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Customize the agent for AI monitoring'
+metaDescription: 'You can apply certain configurations to your APM agents to change how your AI data appears in New Relic.'
+freshnessValidatedDate: never
+---
+
+Once you [install AI monitoring](/install/ai-monitoring), you can configure the default behavior of the agent or update your app to collect different kinds of data. 
+
+## Configure the agent [#configure-agents]
+
+Update default agent behavior for AI monitoring at these agent configuration docs: 
+
+* [Go](/docs/apm/agents/go-agent/configuration/go-agent-configuration/#ai-monitoring)
+* [Node.js](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#ai-monitoring)
+* [Python](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#ai-monitoring)
+* [Ruby](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#ai-monitoring)
+
+## Implement token count API [#enable-token]
+
+<Callout variant="tip">
+If you haven't disabled `ai_monitoring.record_content.enabled`, you don't need to implement the token count callback API.
+</Callout>
+
+Disabling `ai_monitoring.record_content.enabled` stops the agent from sending AI content to New Relic. This means the agent won't send token counts for interactions with your app. You can implement a callback in your app code to determine the token counts locally, then have that information forwarded to New Relic.
+
+Refer to the below snippets as a reference guide for counting tokens locally:
+
+<CollapserGroup>
+    <Collapser
+        id="go-token-api"
+        title="Go"
+    >
+
+        INSERT_TEXT_HERE
+
```

Comment:
If you call `app.SetLLMTokenCountCallback` again with a different callback function value, it will replace the previous one. From that point on, the new callback function will be invoked as needed to provide token counts. Passing `nil` (as in `app.SetLLMTokenCountCallback(nil)` removes any existing callback function, so no function will be called from that point forward until a new one is set again.